lib/ansible/inventory/data.py

```diff
@@ -33,7 +36,235 @@
 display = Display()
 
 
-class InventoryData(object):
+class InventoryHostManager(object):
+    """
+    Manager for inventory host relatives
+    """
+    def host_populate_ancestors(self, host_name, additions=None):
+        """
+        :type host_name: str
+        :type additions: list[str]
+        """
+        host = self.hosts[host_name]
+        if additions is None:
+            for group_name in host.groups:
+                for ancestor_name in self.group_get_ancestors(group_name):
```

Comment:
It looks like `group_get_ancestors` isn't defined in this class but in the group manager class? I'm not sure what the more elegant way to reference one from the other is.